log10/load.py

```diff
@@ -361,42 +388,57 @@ def set_sync_log_text(USE_ASYNC=True):
 
 def log10(module, DEBUG_=False, USE_ASYNC_=True):
     """Intercept and overload module for logging purposes
+    support both openai V0 and V1, and anthropic
 
     Keyword arguments:
     module -- the module to be intercepted (e.g. openai)
     DEBUG_ -- whether to show log10 related debug statements via python logging (default False)
     USE_ASYNC_ -- whether to run in async mode (default True)
 
+    Openai V1 example:
+    Example:
+        >>> from log10.load import log10
+        >>> import openai
+        >>> log10(openai)
```

Comment:
We want this to happen within the class, so we can obsolete the library wrapper, right?

Reply:
remove load.log10 v1 examples here and use load.OpenAI instead (has examples there)